app/controllers/pieces_controller.rb

```diff
@@ -27,4 +26,4 @@ def piece_params
     params.require(:piece).permit(:piece_type, :piece_color, :current_position_x, 
         :current_position_y, :starting_position_x, :starting_position_y)
```

Comment:
`:starting_position_x` and `starting_position_y` do not currently exist as db params. We can clarify the issue in a later pull request.....